packages/embed/src/js/metadata/index.js

```diff
@@ -0,0 +1,236 @@
+import { getFingerprint } from './fingerprint';
+import { parseUserAgent } from './user-agent';
+
+const defaultMetadata = {
+  device_model: null, // string
+  device_os: null, // string
+  browser_name: null, // string
+  browser_version: null, // string
+  document_back_capture_duration_ms: null, // number
+  document_back_capture_retries: null, // number
+  document_back_image_origin: null, // gallery | camera_manual_capture | camera_auto_capture
+  document_back_capture_camera_name: null, // string
+  document_front_capture_duration_ms: null, // number
+  document_front_capture_retries: null, // number
+  document_front_image_origin: null, // gallery | camera_manual_capture | camera_auto_capture
+  document_front_capture_camera_name: null, // string
+  selfie_capture_duration_ms: null, // number
+  selfie_image_origin: null, // gallery | camera_manual_capture | camera_auto_capture
```

Comment:
can you confirm with ope on this too? I think the mobile team uses front_camera and back_camera for the selfie. We do not allow gallery for selfie

Reply:
Here, we only used the value `camera_manual_capture` since there is no other origin. but I'll check with Ope on this. Web devices won't always conform to front/back cameras.

Reply:
We capture from the default camera if we are not using agent mode. But when agent mode is available, we allow the user to switch to the back/other camera's available.